twiml/voice/refer/refer-4/refer-4.7.x.java

```diff
@@ -7,7 +7,7 @@
 public class Example {
     public static void main(String[] args) {
         ReferSip sip = new ReferSip.Builder("sip:alice@example.com").build();
-        Refer refer = new Refer.Builder().action("/handleRefer").method(HttpMethod.POST).referSip(number).build();
+        Refer refer = new Refer.Builder().action("/handleRefer").method(HttpMethod.POST).sip(sip).build();
```

Comment:
Interesting the typing here is different than the C# use case, but unrelated to this PR